website/docs/docs/getting-started.mdx

```diff
@@ -55,7 +55,7 @@ We want to see you succeed with your contract testing journey. Here’s some way
 
 ### Pact Slack
 
-We also have a Slack workspace where you can chat with other members and get general support with contract testing or the product. Join us by registering [here](http://slack.pact.io/).
+We also have a Slack workspace where you can chat with other members and get general support with the contract testing or the product. Join us by registering [here](http://slack.pact.io/).
```

Comment:
```suggestion
We also have a Slack workspace where you can chat with other community members and get general support with contract testing or PactFlow. Join us by registering [here](http://slack.pact.io/).

Please ask PactFlow specific questions in the #pactflow channel (maintained by SmartBear employees), and direct language specific questions to their respective channel such as #pact-jvm for Java or JVM related questions (maintained by the community).
```

Reply:
This was an opportunity to improve some communication around the community supported channels, please feel free to refine.